src/content/docs/nrql/using-nrql/lookups.mdx

```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
 
 import nrqlLookupQueryGeoip from 'images/nrql_screenshot-crop_lookup-query-geoip.webp'
 
-When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries.
-
-## Why use lookup tables? [#why]
-
-For why you'd use lookup tables and how to upload them, see the [Lookup tables overview](/docs/logs/ui-data/lookup-tables-ui/#overview).
+When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries. You can use lookup tables to query data that's not in your New Relic account, help you parse telemetry data, and customize your data groupings.
```

Comment:
Once you've uploaded a CSV to NR isn't it then in your NR account? There's something odd worth clarifying there